data/txHandler.go

```diff
@@ -759,3 +756,17 @@ func (handler *solicitedTxHandler) Handle(txgroup []transactions.SignedTxn) erro
 	}
 	return nil
 }
+
+func handleCapguard(c *util.ErlCapacityGuard, released, served bool) {
```

Comment:
I don't think this function belongs here. Perhaps `erl` should be a sub-package of `util` (similar to `execpool` inside `util`) so that it can use the `logging` package.